test/Microsoft.SqlTools.ManagedBatchParser.IntegrationTests/BatchParser/BatchParserTests.cs

```diff
@@ -338,6 +338,42 @@ public void VerifyIsSqlCmd()
             }
         }
 
+        /// <summary>
+        /// Verify whether the batchParser execute SqlCmd successfully
+        /// </summary>
+        [Fact]
+        public void VerifyRunSqlCmd()
+        {
+            using (ExecutionEngine executionEngine = new ExecutionEngine())
+            {
+                string sqlCmdQuery = @"
```

Comment:
const string sqlCmdQuery, that makes it a compile time constant embeded in generated code.

Reply:
hmm.. I would prefer to use the same practice as used for rest of the tests in this file for consistency unless it breaks something